graphql/queryResolvers/exerciseCrud.ts

```diff
@@ -0,0 +1,78 @@
+import prisma from '../../prisma'
+import {
+  MutationAddExerciseArgs,
+  MutationUpdateExerciseArgs,
+  MutationDeleteExerciseArgs,
+  SuccessResponse
+} from '..'
+import { Context } from '../../@types/helpers'
+import type { Exercise } from '@prisma/client'
+import { isAdmin } from '../../helpers/isAdmin'
+
+export const exercises = async () => {
+  return await prisma.exercise.findMany({
```

Comment:
no need to `return await` just return the promise itself. `return prisma.exercise.findMany....` And take out `async`